.github/workflows/release.yaml

```diff
@@ -51,12 +51,12 @@ jobs:
           set -o pipefail
           set -o nounset
 
-          latest=$(git describe --tags $(git rev-list --tags --max-count=1) 2>/dev/null || echo 0.1.0)
+          latest=$(git describe --tags $(git rev-list --tags --max-count=1) 2>/dev/null || echo api/v0.1.0)
```

Comment:
How will the left side of the `||` operation get the `api/v` prefix?

Reply:
From the previous tag, the `git describe...` will return `api/vX.Y.Z`

Reply:
ack, thanks!